HelloWorldJavascript/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "a9dec65b392646c9a8ab55996c930c7b",
+  "ActionKeyword": "hj",
+  "Name": "Hello World Javascript",
+  "Description": "Hello World Javascript",
+  "Author": "durland",
+  "Version": "1.0.0",
+  "Language": "executable",
```

Comment:
The Flow needs to support the js not in the `exe` method to run the `.js` script.